ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html

```diff
@@ -4,22 +4,44 @@
 {% set notice_blocks = value.blocks_by_name("notice") %}
```

Comment:
Nice digging! I had no idea this method existed.

Reply:
I wish I could take the credit... I literally only changed markup and styling in this PR, and none of the things you commented one where things I had actually changed :)